ding/framework/parallel.py

```diff
@@ -253,24 +253,35 @@ def send_rpc(self, func_name: str, *args, **kwargs) -> None:
             - fn_name (:obj:`str`): Function name.
         """
         if self.is_active:
-            payload = {"f": func_name, "a": args, "k": kwargs}
+            topic = func_name + "::"
+            payload = {"a": args, "k": kwargs}
             try:
-                payload_str = pickle.dumps(payload, protocol=-1)
+                data = pickle.dumps(payload, protocol=-1)
             except AttributeError as e:
                 logging.error("Arguments are not pickable! Function: {}, Args: {}".format(func_name, args))
                 raise e
-            return self._sock and self._sock.send(payload_str)
+            data = topic.encode() + data
+            return self._sock and self._sock.send(data)
 
-    def recv_rpc(self, msg: bytes):
+    def recv_rpc(self, msg: bytes) -> None:
+        """
+        Overview:
+            Recv and parse payload from other processes, and call local functions.
+        Arguments:
+            - msg (:obj:`bytes`): Recevied message.
+        """
+        topic, payload = msg.split(b"::", maxsplit=1)
```

Comment:
add comment about why use topic (such as a link to corresponding communication topology graph)